docs/DeveloperGuide.md

```diff
@@ -133,6 +133,51 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Session Feature
+The add session feature allows users to add individual tuition sessions with specific details of each session.
+
+This section explains the implementation of the add session mechanism and highlights the design considerations 
+taken into account when implementing this feature.
+
+#### Implementation
+The add attendance mechanism is facilitated by `AddAttendanceCommand` and it extends `Command`. The method, 
+`AddSessionCommand#execute()`, performs a validity check on student name input and session details input by the user 
+before adding the session.
+
+The following sequence diagram shows the interactions between the Model and Logic components during the execution of 
+an AddSessionCommand with user input `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`:
+
+![AddSessionCommandSequenceDiagram](images/AddSessionCommandSequenceDiagram.png)
+
+1. `Logic` uses the `AddressBookParser` class to parse the user command.
+2. A new instance of an `AddSessionCommand` would be created by the `AddSessionCommandParser` and returned to AddressBookParser.
```

Comment:
Hi, just wondering, since the AddressBookParser is in `` in the next line, we could possibility `AddressBookParser ` here for consistency as well.

Reply:
Thanks, good catch!